cli.js

```diff
@@ -28,6 +28,16 @@ yargs
     describe: `Set the log file path. It must be an absolute filepath. ` +
     `By default the logs will stored in a timestamp based log file at "${defaultLogDir}".`
   })
+  .option('o', {
+    alias: 'oldTagName',
+    describe: `The tag name for the old specification file.  If include it ` +
+    `indicates that the old spec file is a readme file`
+  })
+  .option('n', {
+    alias: 'newTagName',
+    describe: `The tag name for the new specification file.  If include it ` +
+    `indicates that the new spec file is a readme file`
+  })
```

Comment:
here should be only global parameters ( i.e. not tied to a command), those should go in the specific command folder. You can check [https://github.com/yargs/yargs/blob/master/docs/advanced.md](https://github.com/yargs/yargs/blob/master/docs/advanced.md) for documentation on how to structure yargs.